client/src/main/java/org/apache/cassandra/sidecar/client/RequestExecutor.java

```diff
@@ -305,26 +267,21 @@ private void applyRetryPolicy(CompletableFuture<HttpResponse> future,
         context.retryPolicy()
                .onResponse(future, request, response, throwable, attempt, retryOnNewHost, (nextAttempt, delay) -> {
             String statusCode = response != null ? String.valueOf(response.statusCode()) : "<Not Available>";
+            SidecarInstance nextInstance = sidecarInstance;
```

Comment:
This can be simplified a bit more:
```suggestion
                        SidecarInstance nextInstance = iterator.hasNext()
                                           ? iterator.next()
                                           : sidecarInstance;

            if (response == null || response.statusCode() != HttpResponseStatus.ACCEPTED.code())
            {
                logger.warn("Retrying stream on {} instance after {}ms. " +
                            "Failed on instance={}, attempt={}, statusCode={}",
                            nextInstance == sidecarInstance ? "same" : "next", delay,
                            sidecarInstance, attempt, statusCode, throwable);
            }
            schedule(delay, () -> streamWithRetries(future, consumer, iterator, nextInstance, context, nextAttempt));
```